modules/client-handler/src/main/java/org/apache/ignite/client/handler/configuration/ClientConnectorSslConfigurationSchema.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.client.handler.configuration;
+
+import org.apache.ignite.configuration.annotation.Config;
+import org.apache.ignite.configuration.annotation.ConfigValue;
+import org.apache.ignite.configuration.annotation.Value;
+import org.apache.ignite.internal.network.configuration.KeyStoreConfigurationSchema;
+
+/** Client connector ssl configuration schema. */
+@Config
+public class ClientConnectorSslConfigurationSchema {
```

Comment:
Looks like there is no way to configure client auth (https://netty.io/4.1/api/io/netty/handler/ssl/ClientAuth.html), let's add this too. Should be disabled by default.